metaflow/current.py

```diff
@@ -220,6 +221,18 @@ def parallel(self):
             node_index=int(os.environ.get("MF_PARALLEL_NODE_INDEX", "0")),
         )
 
+    @property
+    def parameter_names(self):
+        """
+        The names of the Parameters defined in the current flow.
+
+        Returns
+        -------
+        List[str]
+            Parameter names.
+        """
+        return self._parameter_names
+
```

Comment:
On reflection, this doesn't make sense. As `parameter_names` is updated directly by the `_update_env` method, which overwrites this property. This also means that the instance var I added `_parameter_names` is never used